state_machine/applications/flight/lib/logs.py

```diff
@@ -79,3 +113,30 @@ def unpack_beacon(bytes):
             "RSSI_dB": rssi,
             "FEI_Hz": fei,
             }
+
+
+def unpack_system(bytes):
+    (lux_xp, lux_yp, lux_zp,
+     lux_xn, lux_yn, lux_zn) = struct.unpack(system_format, bytes)
+    return {"lux_xp": lux_xp,
+            "lux_yp": lux_yp,
+            "lux_zp": lux_zp,
+            "lux_xn": lux_xn,
+            "lux_yn": lux_yn,
+            "lux_zn": lux_zn,
+            }
+
+def unpack_time(bytes):
+    (tm_min, tm_sec) = struct.unpack(time_format, bytes)
+    return {"minutes": tm_min,
+            "seconds": tm_sec
+            }
+
+def unpack_telemetry(bytes):
+    t = unpack_time(bytes[:4])
+    beacon = unpack_beacon(bytes[4:56])
```

Comment:
You should be able to use [struct.calcsize](https://docs.circuitpython.org/en/latest/shared-bindings/struct/index.html#struct.calcsize) to eliminate the use of magic numbers here. 

Like instead of 4, `struct.calcsize(time_format)`.

Reply:
Updated to use calcsize of the formats.